scripts/build/debug/300-gdb.sh

```diff
@@ -69,11 +69,8 @@ do_debug_gdb_build() {
 
         cross_extra_config=("${extra_config[@]}")
         cross_extra_config+=("--with-expat")
-        # NOTE: DO NOT USE --with-libexpat-prefix (until GDB configure is smarter)!!!
```

Comment:
Isn't absence of `--with-libexpat-prefix` equivalent to `--without-libexpat-prefix`? What is the change in behaviour between these two configurations?

Even if this change is needed, I suggest to keep the comment (since your `--without-libexpat-prefix` only reinforces what this comment states - that `--with-libexpat-prefix` is harmful).